lib/tasks/s3_file_conversion.rake

```diff
@@ -1,47 +1,113 @@
 require 'fileutils'
+require 'open-uri'
 include CarrierWave::MiniMagick
 
 namespace :s3 do
   # Usage:
   #     rake s3:file_conversion
   desc "convert manually uploaded files to carrierwave objects"
-  task file_conversion: :environment do |task, args|
+  task image_conversion: :environment do |task|
     success = []
     errors = []
+
     voice_base = Vendor.find_by uid: "voice_base"
 
     [Collection, Transcript].each do |klass|
```

Comment:
Given that you will run the as a take task I would recommend add this inside a `try`, `catch` block. then you can get rid of all the `puts`

```
try
   # code
rescue => ex
    print_result(<pass the error>) # you will have to update the `print_result` method
end
```